gulpfile.js

```diff
@@ -2,7 +2,13 @@ var gulp = require('gulp');
 var sass = require('gulp-sass');
 var uglify = require('gulp-uglify');
 var rename = require("gulp-rename");
+var yargs = require("yargs");
 
+var tag = yargs.argv.tag;
```

Comment:
I believe this should be within the build task as otherwise gulp will throw an error if you try to run any of the other tasks on their own.